### PR TITLE
Wiki: Create New Pages & Update Some

### LINKED ISSUE #240: List publications and press releases about Gridcoin
It might be a good idea to refer to serious articles / papers that discuss Gridcoin on the website.

For example:

* [Environmentalism in Cryptoanarchism: Gridcoin Case Study](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3131232)
* [A Review of Distributed Supercomputing Platforms Using Blockchain](https://link.springer.com/chapter/10.1007/978-981-15-4218-3_13)
* [Sidecoin: a snapshot mechanism for bootstrapping a blockchain](https://arxiv.org/abs/1501.01039)
* [Distributed pool mining and digital inequalities, From cryptocurrency to scientific research](https://www.emerald.com/insight/content/doi/10.1108/JICES-02-2020-0018/full/html)
* [Effizientes Mining: Kryptowährungen für einen besseren Zweck](https://www.computerbase.de/2018-03/effizientes-mining-gridcoin-steem-solarcoin/)

### PR DESCRIPTION
**Pull request by RoboticMind**
Creates pages about:
* Publications mentioning Gridcoin (closes #240 )
* Staking
* BOINC
* Whitelist process
* Command line arguments

Update:
* Addnode page
* Config file page

Adds mentions to some of these new wiki pages across the site. Adds the page on publication mentioning Gridcoin list to the header and footer

### CODE REVIEW

**Comment by RoboticMind** [comment]
@ShmoogleOsukami @barton2526 can you look over this whenever you get the chance?

File: _includes/_footer.htm [hunk 1]
```diff
@@ -151,6 +151,9 @@ <h6>Main Guides</h6>
                     <li>
                         <a href="/assets/img/grc-bluepaper-section-1.pdf">Gridcoin Staking Bluepaper</a>
                     </li>
+                    <li>
+                        <a href="/wiki/publications.html">Publication Mentioning Gridcoin</a>
```
Comment (ShmoogleOsukami) [inline _includes/_footer.htm:155]: missing an s on publication.

**Review by ShmoogleOsukami — COMMENTED** [review]
Not completely proof read everything but visually and functionally everyone works as intended.

File: _includes/_footer.htm (hunk 1 above)
Comment (RoboticMind) [reply _includes/_footer.htm]: fixed that now

**Pull request merged**